core/src/main/java/org/apache/calcite/util/Unsafe.java

```diff
@@ -49,12 +49,6 @@ public static void clear(StringWriter sw) {
     // Included in this class because StringBuffer is banned.
     sw.getBuffer().setLength(0);
   }
-
-  /** Appends to {@link StringWriter}. */
-  public static void append(StringWriter sw, CharSequence charSequence, int start, int end) {
```

Comment:
I am wondering if I should remove directly the unused (public) method or mark it as deprecated. The class is named Unsafe, and I see no reason why end-users should ever call this method.